storage101.md

```diff
@@ -867,24 +789,24 @@ template: admin
 # PersistentVolume: Dynamic provisioning
 
 .center[
-  <img src="provisioning6.png" width="50%"/><br/>
+  <img src="provisioning8.png" width="50%"/><br/>
 ]
 
+
 ---
 
 template: admin
 
-# PersistentVolume: Dynamic provisioning
+# PersistentVolume: Manual provisioning
 
-.center[
-  <img src="provisioning7.png" width="50%"/><br/>
-]
+- "Brownfield" use case.
+  - Using data of old apps.
 
 ---
 
 template: admin
 
-# PersistentVolume: Dynamic provisioning
+# PersistentVolume Life Cycle: Release
```

Comment:
Are these the right titles?

Reply:
good catch, I set all slides to `PersistentVolume: Dynamic Provisioning` (or Release etc)